modules/learn/pages/clusters-and-availability/xdcr-conflict-resolution.adoc

```diff
@@ -1,49 +1,68 @@
 = XDCR Conflict Resolution
-:description: pass:q[_XDCR Conflict Resolution_ automatically synchronizes document-copies that have been modified in different ways at different locations.]
+:description: pass:q[In an XDCR process, modified documents are copied or replicated from the source bucket (or collection) to the target bucket (or collection).]
 :page-aliases: xdcr:xdcr-conflict-resolution,xdcr:xdcr-timestamp-based-conflict-resolution
 
 [abstract]
 {description}
+If a target document with the same document ID as the source document already exists, the XDCR conflict resolution process determines whether the source document replaces the target document or not.
 
 [#conflicts_and_their_resolution]
 == Conflicts and Their Resolution
 
-A _conflict_ is caused when the source and target copies of an XDCR-replicated document are updated independently of and dissimilarly to one another, each by a local application.
-The conflict must be _resolved_, by determining which of the variants should prevail; and then correspondingly saving both documents in identical form.
-XDCR provides an automated _conflict resolution_ process.
+A conflict occurs when a document in the target bucket (or collection) has the same document ID as that of the source bucket (or collection) during an XDCR process. XDCR provides an automated conflict resolution process. XDCR supports the following two alternative conflict resolution policies:
 
-Two, alternative conflict resolution policies are supported: _sequence-number-based_ (which is the default), and _timestamp-based_.
-Note that _timestamp-based_ conflict resolution is only available in the Enterprise Edition of Couchbase Server.
+* Sequence number-based conflict resolution (This is the default policy). 
+* Timestamp-based conflict resolution.
```

Comment:
Remove lines 14-15.

In this section, just explain:

When a source document is modified, XDCR determines whether this revision of the document should be applied to the target.  This process is called conflict resolution, and it is a fully automated process. Two, alternative conflict resolution policies are supported: sequence-number-based (which is the default), and timestamp-based.